main.nf

```diff
@@ -121,6 +121,7 @@ params.multiqc_config = null
 params.noGVCF = null
 params.noStrelkaBP = null
 params.nucleotidesPerSecond = 1000.0
+params.pon = null
```

Comment:
Can you describe this params in `docs/usage.md` and in the `helpMessage` in `main.nf`

Reply:
And can you print that params somewhere here as well: https://github.com/nf-core/sarek/blob/d3b3b31899ab0822d2486f4f1451ddfc08942ec3/main.nf#L209-L229

Reply:
Done.